src/libraries/Microsoft.Extensions.Primitives/tests/StringSegmentTest.cs

```diff
@@ -1590,7 +1524,7 @@ public void StringSegment_EndsWith_Globalized(string a, string b, StringComparis
 
         private static StringSegment MakePaddedStringSegment(string input)
         {
-            return (input is null) ? new StringSegment() : new StringSegment("xx" + input + "zzz", 2, input.Length);
+            return (input is null) ? new StringSegment(null) : new StringSegment("xx" + input + "zzz", 2, input.Length);
```

Comment:
Why is this being changed?

Reply:
Tried to fix `StringSegment_CompareEqual_Globalized` test. This didn't work, so I'll revert it.
How does `new StringSegment()` compile? `StringSegment` doesn't have parameterless constructor, does it?

Reply:
> How does new StringSegment() compile?

It is a struct. Structs always have a parameterless constructor.